loader_hub/google_drive/base.py

```diff
@@ -122,6 +123,13 @@ def _get_fileids_meta(
             if folder_id:
                 fileids_meta = []
                 query = "'" + folder_id + "' in parents"
+
+                #Add mimeType filter to query
+                if mime_types:
+                    mime_types.append("application/vnd.google-apps.folder") #keep the recursiveness
```

Comment:
it's not obvious to me why you always append this - wouldn't you have duplicate values of `"application/vnd.google-apps.folder"` in mime_types?

Reply:
Yes, you're right. How about checking if the folder mime type is in the list before adding it?
It needs to be in the list, otherwise the query will skip the subfolders making it non-recursive.

Reply:
See the change I made to it

Reply:
yeah that works